stat/running/running.go

```diff
@@ -0,0 +1,74 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package running
+
+import "math"
+
+// Mean helps compute running mean
+type Mean struct {
+	// Decay sets the decay rate. A value of 0.99 means that the effective count
+	// is multiplied by 0.99 for every added value. If Decay is zero, a default
+	// value of 1 is used (the count is not decayed).
+	Decay     float64
+	InitCount float64
+	InitMean  float64
+
+	count float64
+	mean  float64
+}
+
+// Reset resets the counter. The next accumulate will use the
+// initial mean and count.
+func (m *Mean) Reset() {
+	m.count = 0
```

Comment:
I don't think I understand how these are being used. I was going to suggest

```
m.count = m.InitCount
m.mean = m.InitMean
```

but then, the code in `AccumWeighted` doesn't make sense.

Reply:
Done.